test/Microsoft.AspNetCore.Blazor.E2ETest/Tests/ComponentRenderingTest.cs

```diff
@@ -156,6 +156,47 @@ public void CanRenderChildComponents()
             Assert.Equal("somevalue", styledElement.GetAttribute("customattribute"));
         }
 
+        // Verifies we can render HTML content as a single block
+        [Fact]
+        public void CanRenderChildContent_StaticHtmlBlock()
+        {
+            var appElement = MountTestComponent<HtmlBlockChildContent>();
+            Assert.Equal("<p>Some-Static-Text</p>",
+                appElement.FindElement(By.CssSelector("#foo")).GetAttribute("innerHTML"));
```

Comment:
Nitpick: `By.CssSelector("#foo")` could be `By.Id("foo")`

I don't honestly mind whether you change this